modules/admin-ui/src/main/webapp/scripts/modules/events/subresources/controllers/toolsController.js

```diff
@@ -48,7 +48,7 @@ angular.module('adminNg.controllers')
         $scope.openTab = function (tab) {
             $scope.tab = tab;
             if ($scope.tab === "editor") {
-              $scope.area   = "segments";
+              $scope.area = "segments";
```

Comment:
True, I guess, and I admittedly am not yet that acquainted with the exact netiquette of the community, but I think small stylistic fixes should be okay to go with other changes. Kind of like in the boy scout rule: Always leave the code better than you found it.

Otherwise where are those changes going to go? "MH-4242: More style fixes"? :wink: 

At least give me credit for putting stuff like this in its own commit. :stuck_out_tongue:

Reply:
I'm more in favour of "Never touch a running system".

It would be appropriate to create a separate issue and PR for style changes if they don't related to the ticket.
If that seems overkill, consider it as an indication that the change is simply not important enough to be individually addressed. Maybe it would then be better to go through more code and fix some similar style issue vs. having the fixes distributed over a large number of actually unrelated pull requests (but still having not fixed them everywhere).

Anyway, that is not a request to revert that change, but a suggestion to try to focus pull requests on the actual problem rather than including changes on whatever was near the code that needed to be changed.

Reply:
Sure, I see your point and will heed your advice in the future. However, I disagree with the notion that more or less artificial and arbitrary (not meant in a bad way, but in the literal sense) bureaucratic hurdles are an indication of the worth of a change. Sure, **this** particular change does not matter at all, but in general and in the aggregate code quality improvements should be encouraged, not encumbered by heavy weight processes. Plus I don't really see the added value of isolating such changes to their own pull request versus to their own commit.

On the other hand I have to admit that I **do** care about git history cleanliness to a probably unhealthy degree. So, in the annotation tool, I collect changes like this in a `nitpicks` branch to keep them out of the feature branches. But there I also have the freedom to just merge that branch every now and then, without going through any kind of process. Inb4: yes that f*cked me over on occasion, already. :sweat_smile: